src/etc/inc/shaper.inc

```diff
@@ -4511,8 +4507,6 @@ class dnqueue_class extends dummynet_class {
 		
 		/* Limiter patch */
 		$selectedAqm = getAQMs()[$data['aqm']];
-		if (!$selectedAqm)
-			$input_errors[] = gettext("Selected scheduler not recognized: " . $data['sched'] . '.');
```

Comment:
This should still be tested -- Keep the test but do not put the variable in the error message.